lib/confluence/set_ops.es6.js

```diff
@@ -295,24 +293,46 @@ foam.CLASS({
 
   methods: [
     function SET_MINUS(dao, sink) {
+      if (Array.isArray(dao))
```

Comment:
It might be easier to follow/maintain if you handled the single case by wrapping in array. That way, the sink initialization stays in one place.